README.md

```diff
@@ -11,6 +11,9 @@ dp-frontend-renderer
 | ENABLE_COOKIES_CONTROL        | false                 | Enabled user cookie preferences
 | PATTERN_LIBRARY_ASSETS_PATH   |                       | Pattern library location
 | SUPPORTED_LANGUAGES           | [2]string{"en", "cy"} | languages supported
+| GRACEFUL_SHUTDOWN_TIMEOUT     | 5s                    | The graceful shutdown timeout
+|	HEALTHCHECK_INTERVAL          | 10s                   | Time between calls to healthchecks
+|	HEALTHCHECK_RECOVERY_INTERVAL | 1m                    | Timeout to consider a failing healthcheck critical
```

Comment:
should be `HEALTHCHECK_CRITICAL_TIMEOUT`? 

(also replace the tab at the start with a space? - not sure if this makes a difference when rendered)